model/controller/dual_mode_controller.cc

```diff
@@ -1576,6 +1576,22 @@ void DualModeController::WriteSynchronousFlowControlEnable(CommandView command)
           kNumCommandPackets, ErrorCode::SUCCESS));
 }
 
+void DualModeController::SetControllerToHostFlowControl(CommandView command) {
```

Comment:
Can I ask you to implement host flow control ?
If you only added the command because zephyr was invoking it, another option would be to remove the command Set Controller to Host Flow Control from the supported command mask in the controller properties.

Reply:
I found that I should just apply the same `CONFIG_BT_HCI_ACL_FLOW_CONTROL=n` as for Android NetSim (supposedly the same Rootcanal?) to the samples. Discarded that commit now

Will make sure I document it when I write a doc for running Zephyr with Rootcanal for Zephyr docs